sympy/core/sympify.py

```diff
@@ -50,6 +52,23 @@ class CantSympify(object):
     """
     pass
 
+# Support for basic numpy datatypes
+
+def convert_numpy_to_sympy(a):
+    np=import_module('numpy')
```

Comment:
Someone can call this with a non-numpy object (?).  I think we should assert np is non-None here, as a sanity check